python/lsst/meas/astrom/astrometry.py

```diff
@@ -213,24 +243,58 @@ def solve(self, exposure, sourceCat):
 
         sourceSelection = self.sourceSelector.run(sourceCat)
 
+        doWriteFits = False
+        if doWriteFits:
+            dataDir = "/sdf/data/rubin/user/laurenma/tickets/DM-30993/"
+            # dataDir = "/sdf/data/rubin/user/laurenma/tickets/DM-46014/"
+            sourceSelection.sourceCat.writeFits("{}srcCat_selected.fits".format(dataDir))
+
```

Comment:
This looks like test code that should be removed.

Reply:
Garhhh...totally forgot to remove the debugging stuff...thanks for the catch!